kubernetes-deployment/bootstrap.py

```diff
@@ -96,6 +96,8 @@ def sftp_paramiko(src, dst, filename, host_config):
     username = host_config['username']
     password = host_config['password']
     port = 22
+    if (host_config['hostport']):
```

Comment:
why do we need to make ssh port in a host env configuration?

Reply:
some env may not allow ssh to use 22 port